### DISCUSSION TITLE
Is there a way to get a random record and/or order a query randomly with SQLmodel?

### DISCUSSION THREAD

**Discussion by bolinocroustibat** [fastapi/sqlmodel#1490] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
random_user = select(User).order_by('?').limit(1)
```

### Description

Is there a way to get a random record and/or order a query randomly, like an equivalent to Django `.order_by('?')` or to SQLalchemy `.order_by(func.rand())` for MySQL?

### Operating System

Linux

### SQLModel Version

0.0.6

### Python Version

3.9

**Comment by mgurg** [comment, marked as answer]
Try this:
```python
from sqlalchemy import func

task = session.exec(select(Tasks).order_by(func.random())).first()
```

**Comment by johnziebro** [comment]
I ended up here looking at how to find the last record. Docs and issues don't have info on this. Thanks for the direction.

Using the code from this issue finding the last record (based on the primary key), you just have to sort the results in descending order with sqlalchemy imports and return first as well:

```python
from sqlalchemy import asc, desc

task = session.exec(select(Tasks).order_by(desc(Tasks.id)).first()
```
